app/[locale]/ledger/income/edit/[idx]/page.tsx

```diff
@@ -14,17 +14,17 @@ export default function EditIncome({ params }: { params: { idx: number } }) {
     const router = useRouter()
     const item = useAppSelector(state => selectIncomeItemAt(state, params.idx))
 
-    function editIncomeClicked({name, description, amount}: IncomeItemFormData) {
+    function editIncomeClicked({description, business, taxesFiled}: IncomeFormJobData) {
         dispatch(setIncomeItem({
             item: {
-                name,
                 description,
-                amount,
-            } as IncomeItem,
+                business,
+                taxesFiled
+            } as JobItem,
             idx: params.idx,
         }))
 
-        router.push('/ledger/income/list')
+        router.push(`/ledger/income/${params.idx}/payment/add`)
```

Comment:
When editing a job the user should probably be redirected to the income list screen not the payment add screen.

Reply:
at this point for sure.